src/ui/App.css

```
@@ -10,6 +10,7 @@
   position: relative;
   display: flex;
   flex-direction: column;
+  max-height: calc(100vh - 50px - 30px);
```

```suggestion
  max-height: calc(100vh - 80px);
```

should give the same result ?
assuming we know the size of the footer through a variable, it could be good to use that one here.